site/ko/README.md

```diff
@@ -7,9 +7,9 @@
 [docs-ko@tensorflow.org](https://groups.google.com/a/tensorflow.org/forum/#!forum/docs-ko)로
 메일을 보내주시기 바랍니다.
 
-노트: [site/en/r2](https://github.com/tensorflow/docs/tree/master/site/en/r2)
+노트: [site/en](https://github.com/tensorflow/docs/tree/master/site/en)
```

Comment:
Can you switch this "Note: ..."

This is a style element used on tensorflow.org for callout boxes. SInce this section from the readme gets copy-pasted into notebooks, I'd rather it be in the correct format by default.)

Reply:
Sure, thank you for your review